docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx

```diff
@@ -0,0 +1,172 @@
+* Ed-FI ODS Admin Api
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![SingleTenant Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
+  </TabItem>
+  <TabItem value="SQL Server" label="SQL Server">
+    ![SingleTenant Configuration - SQL Server](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+  </TabItem>
+</Tabs>
+
+### SingleTenant with ODS Context Configuration
+
+This configuration shows a basic example of _explicit_ data segmentation
+strategy based on school year, i.e. school year becomes a required part of the
+API route segments. See [Context-Based Routing for Year-Specific
+ODS](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559277/Context-Based+Routing+for+Year-Specific+ODS)
+for more information.
+
+Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-FI ODS Admin Api
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![SingleTenant with ODS Context Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenantContext-PostgreSql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenantContext-PostgreSql.png)
+  </TabItem>
+</Tabs>
+
+### MultiTenant Configuration
+
+This configuration shows a basic example of multiple tenant configuration, with
+one Admin, Security, ODS databases per tenant. See [Multi-Tenant
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559218/Single+and+Multi-Tenant+Configuration)
+for more information.
+
+Includes the following web applications:
+
+* Ed-Fi Web API
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![MultiTenant Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/MultiTenant-PostgreSql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/MultiTenant-PostgreSql.png)
+  </TabItem>
+</Tabs>
+
+### MultiTenant with ODS Context Configuration
+
+This configuration shows a basic example of multi tenant configuration with
+_explicit_ data segmentation strategy based on school year. Deployment has one
+Admin, Security per tenant and has one ODS per school year. ODS for the school
+year is selected based on the schoolyear in the API route segment. See
+[Multi-Tenant
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559218/Single+and+Multi-Tenant+Configuration)
+and [Context-Based Routing for Year-Specific
+ODS](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559277/Context-Based+Routing+for+Year-Specific+ODS)
```

Comment:
update this link to reference/ods-api/platform-dev-guide/configuration/context-based-routing-for-year-specific-ods